www/build.sh

```diff
@@ -19,6 +19,18 @@ rm -rf build/
 cp -r public/ build/
 mkdir build/wip # for WIP site
 
+# download the latest code for the examples
+echo 'Downloading latest examples...'
+curl -fLJO https://github.com/roc-lang/examples/archive/refs/heads/main.zip
+unzip examples-main.zip
+cp -R examples-main/examples/ wip_new_website/content/examples/
+
+# relace links in wip_new_website/content/examples/index.md to work on the WIP site
+sed -i 's|](/|](/wip/examples/|g' wip_new_website/content/examples/index.md
+
+# clean up examples artifacts
+rm -rf examples-main examples-main.zip
+
```

Comment:
This downloads [roc-lang/examples](https://github.com/roc-lang/examples) and puts them into the content folder. These will then get built alongside the rest of the site content. I added to ignore so they aren't checked in.

You should run `bash www/build.sh` at least once so you have these files locally before running `bash www/wip_new_website/build-dev-local.sh`